bin/docserv-createconfig

```diff
@@ -267,6 +267,21 @@ def check_langs(langs, message, attempts = 3):
     sys.exit("Expected one or more of these:\n"
         "{}".format(", ".join(languages)))
 
+def check_productid(productid, message, attempts = 3):
+    productid_regex = r'(?P<productid>[-_.+a-zA-Z0-9]+)'
+    message_expected = ("Following chars accepted: -_.+a-zA-Z0-9")
+    while attempts:
+        if not productid:
+            productid = input(message)
+        match = re.match(productid_regex, productid)
+        if match and match.group('productid') == productid:
+            return productid
+        else:
+            attempts -= 1
+            print("Invalid Product ID '{}' given".format(productid))
```

Comment:
I'd suggest to give the user a hint what did he wrong and how to avoid wrong input. This line depends a bit on the outcome from my last comment.

```suggestion
            print("Invalid Product ID {!r} given".format(productid))
            print("Use ...") # Should we add the regex from above here?
```

Reply:
I agree, though instead of the regex I'd suggest the slightly more human-friendly wording "Only alphanumeric characters and the special characters -_.+ are allowed."